.github/workflows/build.yml

```diff
@@ -0,0 +1,14 @@
+name: Java/Maven CI triggering CD-GitOps
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  Build:
+    uses: minova-afis/aero.minova.github.workflows/.github/workflows/java-continuous-integration.yml@main
+    secrets: inherit
```

Comment:
Ersetzt `.github/workflows/continuous-integration.yml`

Reply:
Wenn ich mich recht erinnere, können in public Repos nicht private Workflows verwendet werden.

Ich habe ein public Repo für public Workflows erstellt: https://github.com/minova-afis/aero.minova.os.github.workflows

Da kann man eine Kopie der privaten Workflows einfügen.
Dabei achten, dass da keine Geheimnisse sind, was bisher meines wissen nach nicht der Fall ist.

So wie wir es abgemacht haben: https://github.com/minova-afis/aero.minova.github.workflows/issues/13#issuecomment-1536140987